.projenrc.ts

```diff
@@ -1,6 +1,6 @@
 import {javascript, cdk} from 'projen'
 
-const repository = 'https://github.com/AminFazlMondo/Lerna-Projen.git'
+const repository = 'https://github.com/Dazza65/Lerna-Projen.git'
```

Comment:
Could you please revert the changes regarding the use of your user name?

Reply:
Done.

Reply:
There are more references to your user name in some workflow files and package.json

Reply:
Should just be the 3 files changed now: src/index.ts, src/types.ts and test/index.test.ts